{"uuid": "e5114bdc-d0aa-4fe1-910c-a1a4a2cfa203", "children": ["b86ed737-406f-4560-b9d9-22ea0f0b6a38"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'desired_capabilities'\n", "trace": "  File \"C:\\Users\\Ascendion\\Downloads\\converted_pytest_browserstack\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\Ascendion\\Downloads\\converted_pytest_browserstack\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"C:\\Users\\Ascendion\\Downloads\\converted_pytest_browserstack\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Ascendion\\Downloads\\converted_pytest_browserstack\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Ascendion\\Downloads\\converted_pytest_browserstack\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Ascendion\\Downloads\\converted_pytest_browserstack\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Ascendion\\Downloads\\converted_pytest_browserstack\\conftest.py\", line 31, in driver\n    driver = Remote(command_executor=BROWSERSTACK_URL, desired_capabilities=caps)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1761727939983, "stop": 1761727939983}], "start": 1761727939983, "stop": 1761727940054}